rules/CodingStyle/Rector/ClassConst/RemoveFinalFromConstRector.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\ClassConst;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\ClassConst;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Privatization\NodeManipulator\VisibilityManipulator;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @changelog https://php.watch/versions/8.1/final-class-const
+ *
+ * @see \Rector\Tests\CodingStyle\Rector\ClassConst\RemoveFinalFromConstRector\RemoveFinalFromConstRectorTest
+ */
+final class RemoveFinalFromConstRector extends AbstractRector
```

Comment:
the rule needs to implements `Rector\VersionBonding\Contract\MinPhpVersionInterface` so you can add:

```php
    public function provideMinPhpVersion(): int
    {
        return \Rector\Core\ValueObject\PhpVersionFeature::FINAL_CLASS_CONSTANTS;
    }
```

so it ensure check on php 8.1

Reply:
```suggestion
final class RemoveFinalFromConstRector extends AbstractRector implements \Rector\VersionBonding\Contract\MinPhpVersionInterface
```